senza/templates/postgresapp.py

```diff
@@ -243,7 +245,6 @@
           - Effect: Allow
             Action:
               - "kms:Decrypt"
-              - "kms:Encrypt"
```

Comment:
See commit dfafff128718dc032500c744e80033a850935af2, it was never used. Trying to apply the principle of least privilege here.